test/fixtures/copy-folder-contents/no-state-files/terraform.tfvars

```diff
@@ -1 +0,0 @@
- terragrunt = {}
```

Comment:
Intentional to delete, same with other .tfvars files?

Reply:
Yes, since we switched to tg19